src/renderer/UpSetCellRenderer.ts

```diff
@@ -22,10 +22,10 @@ export default class UpSetCellRenderer implements ICellRendererFactory {
   }
 
   private static createDOMContext(col: ISetColumn) {
-    const categories = col.categories;
+    const labels = col.labels;
     let templateRows = '';
-    for (const cat of categories) {
-      templateRows += `<div class="${cssClass('upset-dot')}" title="${cat.label}"></div>`;
+    for (const lab of labels) {
```

Comment:
```suggestion
    for (const label of labels) {
```

I buy the two letters and would name it `label`. 😉